timesketch/frontend-ng/src/components/LeftPanel/SigmaRule.vue

```diff
@@ -16,39 +16,87 @@ limitations under the License.
 <template>
   <div>
     <v-row no-gutters class="pa-3" :class="$vuetify.theme.dark ? 'dark-hover' : 'light-hover'">
-      <div @click="expanded = !expanded" style="cursor: pointer; font-size: 0.9em">
+      <div @click="expanded = !expanded" class="SigmaRuleTitle">
         <v-icon v-if="!expanded">mdi-chevron-right</v-icon>
         <v-icon v-else>mdi-chevron-down</v-icon>
         {{ sigmaRule.title }}
+        <v-chip rounded x-small class="mr-2">{{ sigmaRule.status }}</v-chip>
       </div>
       <v-spacer></v-spacer>
-      <div><v-icon>mdi-dots-vertical</v-icon></div>
+      <div>
+        <v-menu offset-y>
+          <template v-slot:activator="{ on, attrs }">
+            <v-avatar>
+              <v-btn small icon v-bind="attrs" v-on="on">
+                <v-icon>mdi-dots-vertical</v-icon>
+              </v-btn>
+            </v-avatar>
+          </template>
+          <v-card>
+            <v-list>
+              <v-list-item-group color="primary">
+                <v-list-item>
+                  <v-list-item-icon>
+                    <v-icon>mdi-brightness-6</v-icon>
+                  </v-list-item-icon>
+                  <div>
+                    <router-link
+                      :to="{ name: 'Studio', params: { id: sigmaRule.rule_uuid, type: 'sigma' } }"
+                      style="text-decoration: none; color: black"
+                      >Edit Rule</router-link
+                    >
+                  </div>
+                </v-list-item>
+                <v-list-item v-on:click="deprecateSigmaRule(sigmaRule.rule_uuid)">
```

Comment:
Deactivate or Deprecate? What does this do?

Reply:
it sets the status of the rule to deprecated, which means it will not be picked up by the Sigma Analyzer

Reply:
Ok, please pick one wording to make it easier to follow the code

Reply:
done